### ISSUE TITLE
[Bug] There are too many Spark sessions and the session timeout parameter is not effective

### ISSUE THREAD

**Issue by A-little-bit-of-data** [apache/kyuubi#6855]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the bug

When using kyuubi to start a spark SQL ON K8S cluster, the number of internal sessions increases. The parameter `kyuubi.engine.user.isolated.spark.session.idle.timeout=PT30M` is found in the documentation. The default value is 6H. After changing it to 30M, there are still sessions that exceed 30M and are not closed, and the SQL of the session has been executed. I would like to ask whether the creation of this session is based on a SQL or how it is created. I found that there are not so many SQLs in my cluster, but there are many sessions. I don't know how they are created.

![image](https://github.com/user-attachments/assets/978c1aa4-0d32-4e97-82e1-9ed48e4addb0)

![image](https://github.com/user-attachments/assets/646a9925-47e8-4f78-acb7-ec9d06008922)

### Affects Version(s)

1.9.1

### Are you willing to submit PR?

- [ ] Yes. I would be willing to submit a PR with guidance from the Kyuubi community to fix.
- [ ] No. I cannot submit a PR at this time.

**Comment by pan3793** [comment]
you should read the docs carefully, the docs of  `kyuubi.engine.user.isolated.spark.session.idle.timeout` say 

> If kyuubi.engine.user.isolated.spark.session is false ...

and the default value of `kyuubi.engine.user.isolated.spark.session` is `true`, further, the docs of `kyuubi.engine.user.isolated.spark.session` say

> ... if the engine is running in a group or server share level ... Note that, it does not affect if the share level is connection or user.

**Comment by pan3793** [comment]
for your cases, maybe you should change `kyuubi.session.idle.timeout`

**Comment by holiday-zj** [comment]
kyuubi.session.idle.timeout可以放到创建session的rest请求中，针对某个特定的session设置吗

**Comment by pan3793** [comment]
@holiday-zj no, yet. it's a server-side configuration that the client can not override, but the behavior can be discussed, in a quick thought, we can make changes to support that.

**Comment by shuyouZZ** [comment]
我们支持reset idle timeout吗？好像只在执行某些操作时会刷新lastAccessTime和lastIdleTime
